Automattic/sensei
# Onboarding design tweaks

Based on #5733

### Changes proposed in this Pull Request

* After a talk w/ @Luchadores, we tweaked some onboarding styles. In summary, we made the content and the illustration vertically aligned.
* Notice that I changed from background to image in the illustrations, so we can make it centered, and make sure that the image won't crop on top.

### Testing instructions


* Navigate through the onboarding, and make sure the styles are working properly. `/wp-admin/admin.php?page=sensei_setup_wizard`


### Screenshot / Video

https://user-images.githubusercontent.com/876340/191988642-d823a771-0a52-42e0-a544-13e45f10cd1d.mp4
